src/components/filterSelection/FilterSelection.js

```diff
@@ -63,6 +70,8 @@ const FilterSelection = ({ label, modificationText, applied, status, amountAppli
     <Box
       ref={ref}
       className={cx(theme['select-control'])}
+      role="button"
+      tabIndex={0}
```

Comment:
Q: Would it make sense to just make this element a button instead of a div or did that cause other issues?

Reply:
Tried that as well, but then I have to get rid of the default styling of the button, normally the role="button" will have the same effect for accessibility (aria)